.editorconfig

```diff
@@ -0,0 +1,20 @@
+# Strider Editor/IDE Settings
+# This file is used to promote consistent source code standards
+# amongst all Strider-CD contributors.
+# More information can be found here: http://editorconfig.org/
+
+# General Settings
+root = true
+
+# Settings for all files
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
I only know this from my PHP days.  I cannot remember the exact details of it but our team ran into an issue where this mattered at one point.  We've just kept the setting and I never questioned it.